packages/web/pages/justread/index.tsx

```diff
@@ -145,9 +154,9 @@ const HomeItemView = (props: HomeItemViewProps): JSX.Element => {
       }}
       onClick={(event) => {
         if (event.metaKey || event.ctrlKey) {
-          window.open(props.homeItem.url, '_blank')
+          window.open(`/me/${props.homeItem.slug}`)
```

Comment:
Is there any reason we use `window.open` here?

Reply:
Yeah this is if you are holding down the CTRL key it will open in a new tab.